stdpopsim/slim_engine.py

```diff
@@ -662,6 +652,7 @@ def fix_time(event):
             stdpopsim.ext.validate_time_range(start_time, end_time)
 
         if isinstance(ee, stdpopsim.ext.DrawMutation):
+            # TODO: warning if mutation type id points to a neutral mut.
```

Comment:
I think this TODO would be really heplful to people.

Reply:
Agreed.

Reply:
it turns out this will be annoying to implement. all our tests have neutral mutations specified. we need a follow up issue to do this.